pkg/daemon/feature/feature.go

```diff
@@ -0,0 +1,51 @@
+package feature
+
+/*
+  Copyright 2021 The Rama Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import (
+	"github.com/spf13/pflag"
+	"k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+	"k8s.io/klog"
+)
+
+func init() {
+	if err := feature.DefaultMutableFeatureGate.Add(DefaultRamaFeatureGates); err != nil {
+		klog.Fatalf("feature gate init fails: %v", err)
+	}
+
+	feature.DefaultMutableFeatureGate.AddFlag(pflag.CommandLine)
+}
+
+const (
+	MultiCluster featuregate.Feature = "MultiCluster"
```

Comment:
why not append this featuregate in package `pkg/feature`?

Reply:
fixed, replace pkg/daemon/feature with pkg/feature.